assets/newspack-ui/js/segmented-control/index.js

```diff
@@ -0,0 +1,34 @@
+import { domReady } from '../utils'; // Global utils.
+
+domReady( function () {
+	function segmented_control( element ) {
+		const header = element.querySelector( '.newspack-ui__segmented-control__tabs' );
```

Comment:
Non-blocking nit: can we always assume these child elements will exist inside a segmented control element? (e.g. header, tabs, etc.)

Reply:
@dkoo We should be able to assume the header and tabs will be there, but I'm happy to add checks if it'd be better! 

I discussed this component a bit with @thomasguillot and it won't always have the content/panels, it can just be tabs acting more like a standalone radio component. 

This needs a yay/nay from design still, so I'm happy to make tweaks in the meantime!